fastunit/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
We can just clone fastunit from GitHub in Docker container instead (without creating an image) by adding this command to `Docker` file and use it in pre-submit:
```bash
WORKDIR /fastunit
RUN git clone <fastunit repo> .
```

This way we will not need to create Docker image with pre-backed fastunit and will not need to include it into the init action repo.

Reply:
Done.